### PR TITLE
revise getting started and cli docs

### PR DESCRIPTION
**Pull request by paulirish**
well i started writing out #49 but then  figured id just try out the changes myself. :)

hope you dont hate these!

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -6,76 +6,51 @@ Estimated Time: ~15 minutes
...
 ## Prerequisites
 
-Before you can start using this guide, your project should already meet the following requirements:
+Your project should meet the following requirements:
 
 1. Source code is managed with git (GitHub, GitLab, Bitbucket, etc).
-2. Branches/pull requests are gated on the results of a continuous integration build process (Travis CI, CircleCI, Azure Pipelines, AppVeyor, etc).
+2. Branches/pull requests are gated on the results of a continuous integration build process (Travis CI, CircleCI, Azure Pipelines, AppVeyor, GitHub Actions, etc).
 3. Your CI process can build your project into production assets (typically provided as an `npm run build` command by most frameworks).
+4. Your project either A) has a command that runs a production-equivalent web server, or B) is a static site.
```
Comment (patrickhulce) [inline docs/getting-started.md]:
```suggestion
4. Your project either:
    A) has a command that runs a production-equivalent web server.
    B) is a static site.
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -6,76 +6,51 @@ Estimated Time: ~15 minutes
...
-  - 'if [ "${TRAVIS_NODE_VERSION}" = "8" ]; then lhci autorun ; fi' # only run LHCI on node 8
+  - npm test      # run your normal tests
+  - lhci autorun  # run lighthouse CI against your static site
 ```
 
-**NOTE: for sites with an API or other dynamic component, you'll need to tell LHCI how to start your server**
+That's it! With this in place, you'll have Lighthouse reports collected and and asserted.
 
-With node in your `package.json`...
+See the [autorun docs](./cli#autorun) for more including running your own webserver and uploading the reports for later viewing.
```
Comment (patrickhulce) [inline docs/getting-started.md]: good call, I didn't like how complicated the quick start was getting :)

File: docs/getting-started.md [hunk 3]
```diff
@@ -147,25 +122,27 @@ fi
...
+To run Lighthouse CI, the built site you'd like to test with Lighthouse needs to be available on a server. You can either use the built-in Lighthouse CI static server, a custom local development server, or deploy to a staging location that's accessible to your CI.
+
+For this example, we'll assume your site is already built in a local directory called `./dist`, and we'll use the the `http-server` node package as an example custom server implementation (do not follow this structure just to use `http-server`, it is less fully featured than `lhci collect --static-dist-dir=./dist`, see [Run Lighthouse CI](#run-lhci) for more).
```
Comment (patrickhulce) [inline docs/getting-started.md]:
```suggestion
For this example, we'll assume your site is already built in a local directory called `./dist`, and we'll use the the `http-server` node package as an example custom server implementation (do not follow this structure just to use `http-server`, it is less fully featured than `lhci collect --static-dist-dir=./dist`, see [Run Lighthouse CI](#run-lighthouse-ci) for more).
```

File: docs/getting-started.md [hunk 4]
```diff
@@ -216,91 +193,58 @@ The setup so far will automatically assert the Lighthouse team's recommended set
...
+### Saving Reports to Temporary Public Storage
 
-There's a free service that provides temporary public storage of your Lighthouse reports so you can examine the HTML report of any build without tokens or extra infrastructure. Just add a single `lhci upload` command _after_ `lhci assert`.
+While the Lighthouse CI server offers the complete LHCI experience, you do have to host it yourself. If you'd like basic storage so you can just get a URL for each report, we offer a free service that provides temporary public storage.
 
-**NOTE: as the name implies this is _temporary_ and _public_ storage, do not use if you're uncomfortable with the idea of your Lighthouse reports being stored on a public URL on Google Infrastructure. Reports are automatically deleted 7 days after upload.**
+**NOTE: as the name implies this is _temporary_ and _public_ storage, do not use if you're uncomfortable with the idea of your Lighthouse reports being stored on a public URL on Google Cloud Infrastructure. Reports are automatically deleted 7 days after upload.**
```
Comment (patrickhulce) [inline docs/getting-started.md]:
```suggestion
**NOTE: as the name implies this is _temporary_ and _public_ storage, do not use if you're uncomfortable with the idea of your Lighthouse reports being stored on a public URL on Google Cloud infrastructure. Reports are automatically deleted 7 days after upload.**
```

File: docs/getting-started.md [hunk 5]
```diff
@@ -216,91 +193,58 @@ The setup so far will automatically assert the Lighthouse team's recommended set
...
 
-Historical reports and advanced report diffing is available on the Lighthouse CI server. For setup of the server itself, see [our server recipe](./recipes/README.md). Once the server is setup, you can use the wizard to create a project.
+Once the server is set up, create a a new project with the lhci wizard:
 
 ```bash
 $ lhci wizard
 ? Which wizard do you want to run? new-project
 ? Which server would you like to use? http://url-of-your-lhci-server.com/
 ? What would you like to name the project? My Favorite Project
 ? Where is the project built? https://travis-ci.org/GoogleChrome/lighthouse-ci
```
Comment (patrickhulce) [inline docs/getting-started.md]:
```suggestion
? Where is the project's code hosted? https://github.com/GoogleChrome/lighthouse-ci
```

File: docs/getting-started.md [hunk 6]
```diff
@@ -216,91 +193,58 @@ The setup so far will automatically assert the Lighthouse team's recommended set
...
-With your token, you can configure your Lighthouse CI integration to upload reports to the server.
-
-```bash
-#!/bin/bash
-
-# ...
-
-lhci assert --rc-file=lighthouserc.json
-EXIT_CODE=$?
+Add your token to your CI with the environment variable `LHCI_TOKEN`. Alternatively, you can pass it with to `upload` with the `--token` flag:
```
Comment (patrickhulce) [inline docs/getting-started.md]:
```suggestion
Add your token to your CI with the environment variable `LHCI_TOKEN`. Alternatively, you can pass it to `upload` with the `--token` flag:
```

File: packages/cli/src/collect/collect.js [hunk 7]
```diff
@@ -22,7 +22,9 @@ function buildCommand(yargs) {
     method: {type: 'string', choices: ['node'], default: 'node'},
     headful: {type: 'boolean', description: 'Run with a headful Chrome'},
     additive: {type: 'boolean', description: 'Skips clearing of previous collect data'},
-    url: {description: 'The URL to run Lighthouse on.'},
+    url: {
+      description: 'A URL to run Lighthouse on.  You can evaluate multiple URLs by adding this flag multiple times.'
```
Comment (patrickhulce) [inline packages/cli/src/collect/collect.js]:
```suggestion
      description: 'A URL to run Lighthouse on. You can evaluate multiple URLs by adding this flag multiple times.',
```

File: packages/cli/src/collect/collect.js [hunk 8]
```diff
@@ -22,7 +22,9 @@ function buildCommand(yargs) {
     method: {type: 'string', choices: ['node'], default: 'node'},
     headful: {type: 'boolean', description: 'Run with a headful Chrome'},
     additive: {type: 'boolean', description: 'Skips clearing of previous collect data'},
-    url: {description: 'The URL to run Lighthouse on.'},
+    url: {
+      description: 'A URL to run Lighthouse on. You can evaluate multiple URLs by adding this flag multiple times.',
```
Comment (patrickhulce) [inline packages/cli/src/collect/collect.js]:
```suggestion
      description: 
        'A URL to run Lighthouse on. You can evaluate multiple URLs by adding this flag multiple times.',
```

**Comment by patrickhulce** [comment]
@googlebot I consent.

File: docs/getting-started.md (hunk 5 above)
Comment (paulirish) [reply docs/getting-started.md]: oh really? i thought this question was asking about your travis URL.

oh you changed it. neat :)

**Comment by paulirish** [comment]
awesome thanks. real happy you liked this

**Pull request merged**